package.json

```diff
@@ -90,6 +91,7 @@
     "eslint-plugin-react": "7.10.0",
     "identity-obj-proxy": "3.0.0",
     "jest": "23.4.1",
+    "jest-canvas-mock": "^1.1.0",
```

Comment:
pin ^^
Maybe add in a `.yarnrc` file to pin deps by default?

Reply:
Didn't know that was an option for pinning these when installing, I'll look into it!